src/PrefsManager.h

```diff
@@ -57,26 +57,16 @@ enum ShowDancingCharacters
 	NUM_ShowDancingCharacters,
 	ShowDancingCharacters_Invalid
 };
-enum BannerCacheMode
+enum ImageCacheMode
```

Comment:
Using an enum class makes this more type safe but it's not that important.

Reply:
most of the stuff in prefs use it so I'ma leave it in for now, if we are planning to remove it entirly lets remove it then 👍